test/language/comments/hashbang-encoded-bang.js

```diff
@@ -0,0 +1,10 @@
+#\u0021
+/*---
+esid: pending
+description: >
+    Hashbang comments should not be allowed to have encoded characters
+info: |
+    HashbangComment::
+      #! SingleLineCommentChars[opt]
+flags: [raw]
```

Comment:
Could you please add `feature` front matter to each test?

```
features: [hashbang]
```

presumably

Reply:
(and update [`features.txt`](https://github.com/tc39/test262/blob/master/features.txt) accordingly)